topology/mask.h

```diff
@@ -250,8 +255,25 @@ class BoxMask : public Mask< D >
   static Name get_name();
 
 protected:
+  /**
+   *  Calculate the min/max x, y, z values in case of a rotated box.
+   */
+  void calculate_min_max_values_();
+
   Position< D > lower_left_;
   Position< D > upper_right_;
+
+  Position< D > min_values_;
```

Comment:
Could you add a comment what these are and how they relate to lower_left/upper_right?

Reply:
@heplesser I have added the comment. The min/max values correspond to the lower_left/upper_right of the boundary box around the rotated box. Should I change the names to something like `bbox_lower_left_` and `bbox_upper_right_` to make it more clear?